tests/FINDOLOGIC/Export/Tests/XmlSerializationTest.php

```diff
@@ -369,6 +369,12 @@ public function urlValidationProvider()
         ];
     }
 
+    /**
+     * @dataProvider urlValidationProvider
+     *
+     * @param string $value
+     * @param \Exception|null $expectedException
+     */
     public function testUrlValidationWorks($value = '', $expectedException = null)
```

Comment:
Defaults can be removed now. This way we also ensure that the test breaks if we remove the data provider again in the future by accident.

Reply:
Done and updated the test because calling the URL class was wrong